docs/4-user-guide/2-zarf-packages/3-the-zarf-init-package.md

```diff
@@ -4,36 +4,41 @@ sidebar_position: 3
+The init package is the `zarf.yaml` file that lives at the [root of the Zarf repository](https://github.com/defenseunicorns/zarf/blob/main/zarf.yaml).
+It is defined by composed components which provide a foundation for future packages to utilize. Upon deployment, the init package generates a `zarf` namespace within your K8s cluster and deploys pods, services, and secrets to that namespace based on the optional components selected for deployment.
 
 ## Mandatory Components
 
-Zarf's work necessitates that some components in the [init package](https://github.com/defenseunicorns/zarf/blob/main/zarf.yaml) are "always on" (a.k.a. required & cannot be disabled). These components are always deployed whenever you perform a `zarf init` command. Those include:
+Zarf's capabilities require certain components of the init package to be constantly active, meaning that they cannot be disabled and are always on. These components are automatically deployed whenever a `zarf init` command is executed. These components include:
 
-|                         | Description                                                                                                          |
+| Components              | Description                                                                                                          |
 | ----------------------- | -------------------------------------------------------------------------------------------------------------------- |
 | zarf-injector           | Adds a Rust and Go binary to the working directory to use during the registry bootstrapping.
 | container-registry-seed | Adds a container registry so Zarf can bootstrap itself into the cluster.                                             |
 | container-registry      | Adds a container registry service&mdash;[docker registry](https://docs.docker.com/registry/)&mdash;into the cluster. |
 
 ## Additional Components
 
-In addition to those that are always installed, Zarf's optional components provide additional functionality and can be enabled as & when you need them.
+In addition to required components, Zarf also offers optional components that provide additional functionality and can be enabled as needed. 
 
-These optional components for the init package are listed below along with the "magic strings" you pass to `zarf init --components` to pull them in:
+Below are the optional components available for the init package, along with their respective "magic strings" that can be passed to `zarf init --components` to pull them in:
```

Comment:
```suggestion
Below are the optional components available for the init package, along with their respective component names that can be passed to `zarf init --components` to deploy them:
```

Might be better to say component names rather than magic strings.